initial commit

Katsiff opened the pull request:
Working on this file still. Tragically lost my entire file last week and
had to start over. Still getting exception errors. I have read through
the book,, tried using different pieces of their code logic. Also gone
through the Oracle tutorial.  I think I have confused several items and
need to take a break. Try again later. Saving for prosperity.

Katsiff committed:
initial commit

Working on this file still. Tragically lost my entire file last week and
had to start over. Still getting exception errors. I have read through
the book,, tried using different pieces of their code logic. Also gone
through the Oracle tutorial.  I think I have confused several items and
need to take a break. Try again later. Saving for prosperity.

Katsiff committed:
Getting closer....

will work on more later.

ninjapanzer commented on src/assignment/pkg9/DiceTower.java:
```diff
@@ -0,0 +1,70 @@
+    
+});
+}
+}
```
Indentation went funny here

Katsiff committed:
Fixed indentation

When I went back to redo my code, my indentation must have gotten out of
whack. Quick fix this morning.

Zammy95 commented:
Really nice code. Its organized well, and your comments help a lot in figuring out how it works.
